src/loading-bar.css.ts

```diff
@@ -14,17 +14,21 @@ const enterAnimation = keyframes({
     },
 });
 
+export const portalAnimation = style({
+    animation: `${enterAnimation} ${TRANSITION_DURATION_MS}ms ease-out`,
+    transition: `transform ${TRANSITION_DURATION_MS}ms ease-out`,
+});
+
 export const portal = style([
     sprinkles({
         position: 'fixed',
         top: 0,
         left: 0,
         right: 0,
+        isolation: 'isolate',
     }),
     {
-        zIndex: 9999,
-        animation: `${enterAnimation} ${TRANSITION_DURATION_MS}ms ease-out`,
-        transition: `transform ${TRANSITION_DURATION_MS}ms ease-out`,
+        zIndex: 1, // Render above everything, including other portals
```

Comment:
LoadingBar requires zIndex despite being rendered inside a portal, because we don't want it to be covered by other portals